### PR TITLE
Complete javascript-carousel exercise

### PR DESCRIPTION
**Pull request by BenGrimmius**

### CODE REVIEW

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]
@BenGrimmius Please include a video with your pull request that demonstrates the carousel's functionality

**Comment by BenGrimmius** [comment]
[Kapture 2022-12-12 at 19.48.06.gif.zip](https://github.com/BenGrimmius/m2-1222-code-solutions/files/10214090/Kapture.2022-12-12.at.19.48.06.gif.zip)

File: javascript-carousel/main.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  } else if (dotFour.className === 'dot four fa-solid fa-circle') {
+    img.setAttribute('src', 'images/039.png');
+    dotFour.setAttribute('class', 'dot four fa-regular fa-circle');
+    dotFive.setAttribute('class', 'dot five fa-solid fa-circle');
+  } else if (dotFive.className === 'dot five fa-solid fa-circle') {
+    img.setAttribute('src', 'images/001.png');
+    dotFive.setAttribute('class', 'dot five fa-regular fa-circle');
+    dotOne.setAttribute('class', 'dot one fa-solid fa-circle');
+  }
+}
```
Comment (TimelikeClosure) [inline javascript-carousel/main.js]: This is a _lot_ of code repetition. Please attempt to combine some of this in ways that let you re-use code between slides.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by BenGrimmius** [comment]
Kapture 2022-12-15 at 19 14 35

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**